icu4c/source/test/intltest/tsmthred.cpp

```diff
@@ -777,7 +763,7 @@ void MultithreadTest::TestThreadedIntl()
     UErrorCode threadSafeErr = U_ZERO_ERROR;
 
     ThreadSafeFormatSharedData sharedData(threadSafeErr);
-    assertSuccess("initializing ThreadSafeFormat", threadSafeErr, TRUE);
+    assertSuccess(WHERE, threadSafeErr, TRUE);
```

Comment:
what is `WHERE`?

Reply:
It's a macro in the test framework for the source file and line number. Makes a nice message for test failures.